http-client/src/test/java/io/airlift/http/client/TestingHttpServer.java

```diff
@@ -85,7 +85,7 @@ public TestingHttpServer(Optional<String> keystore, Servlet servlet, Consumer<Ht
 
         ServletHolder servletHolder = new ServletHolder(servlet);
         ServletContextHandler context = new ServletContextHandler(ServletContextHandler.NO_SESSIONS);
-        context.setGzipHandler(new GzipHandler());
+        context.insertHandler(new GzipHandler());
```

Comment:
It would be better to put `GzipHandler` outside of the `ServletContextHandler`.

In other words, the `GzipHandler` wraps `ContextHandlerCollection`.

Reply:
Thanks, updated. Should we make a similar change in `HttpServer`?

Reply:
I didn't notice the behavior in HttpServer, but yes, it should change to match this pattern.